lib/shopify_cli/theme/theme.rb

```diff
@@ -24,7 +24,8 @@ def theme_files
       end
 
       def static_asset_files
-        glob("assets/*").reject(&:liquid?)
+        liquid_assets = glob("assets/*.liquid").map { |file| "#{file.relative_path}" }
```

Comment:
I'm looping @macournoyer to get his input here because he has more context here.